vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSequencesController.java

```diff
@@ -63,6 +63,11 @@ public String sequence(Model model, @PathVariable("sequenceId") String sequenceI
             throw new SlidesInSequenceNotFoundException();
         }
         String firstSlideId = slides.get(0).getId();
-        return "redirect:/exhibit/{spaceId}/module/" + moduleId + "/sequence/" + sequenceId + "/slide/" + firstSlideId +"?branchingPoint="+branchingPointId+"&previousSequenceId="+previousSequenceId;
+        
+        if(branchingPointId==null) {
+            return "redirect:/exhibit/{spaceId}/module/" + moduleId + "/sequence/" + sequenceId + "/slide/" + firstSlideId;
+        }else {
+            return "redirect:/exhibit/{spaceId}/module/" + moduleId + "/sequence/" + sequenceId + "/slide/" + firstSlideId +"?branchingPoint="+branchingPointId+"&previousSequenceId="+previousSequenceId;
+        }
```

Comment:
how about 
```
...?branchingPoint="+(branchingPointId != null ? branchingPointId : "") + ...
```
or even better use `String.format`

Reply:
If we do this, the null check will again fail in the slide controller because this will lead parameter to take "" value in slide controller.

Reply:
Let me check how it works if i pass null to slideController for null in sequenceController